src/main/java/org/amahi/anywhere/adapter/ServerFilesAdapter.java

```diff
@@ -104,8 +109,28 @@ private View newView(ViewGroup container) {
 	private void bindView(ServerFile file, View view) {
 		ImageView fileIconView = (ImageView) view.findViewById(R.id.icon);
 		TextView fileTextView = (TextView) view.findViewById(R.id.text);
+		TextView fileSize = (TextView) view.findViewById(R.id.file_size);
+		TextView fileLastModified = (TextView) view.findViewById(R.id.last_modified);
+               LinearLayout moreInfo = (LinearLayout) view.findViewById(R.id.more_info);
 
 		fileTextView.setText(getFileName(file));
+
+               int size=getFileSize(file);
+
+               if(size==0){
+                 moreInfo.setVisibility(View.GONE);
```

Comment:
@csoni111 I think for distinguishing a single file type from a folder, size can be used as a measure. Also displaying 0B below a folder doesn't seem reasonable. So may be hiding size is better than displaying size as 0B below folder name.